packages/xstate-solid/src/deepClone.ts

```diff
@@ -17,6 +17,7 @@ export function isWrappable(obj: any): obj is object {
  * @param valueRefs A WeakMap that stores a reference from the original
  *   object/array to the cloned object/array
  */
+// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-constraint
 const clone = <T extends unknown>(
```

Comment:
would this work instead?
```suggestion
const clone = <T,>(
```